pkg/operator/controller/certificate/default_cert.go

```diff
@@ -17,67 +17,73 @@ import (
 
 // ensureDefaultCertificateForIngress creates or deletes an operator-generated
 // default certificate for a given IngressController as appropriate.  Returns true
-// if it creates, updates, or deletes the secret for the certificate, false
+// if it creates or deletes the secret for the certificate, false
 // otherwise.
```

Comment:
For other `ensure` functions, the Boolean indicates whether the object exists, not whether it was just created.  Any reason `ensureDefaultCertificateForIngress` should be different?  (This question concerns the logic as it was prior to this PR, so feel free to defer on this point in this PR.)

Reply:
No wonder this file was confusing to work with. I've adjusted the return values to match our other `ensure` functions, as you suggested.